.ci/Jenkinsfile

```diff
@@ -25,6 +25,7 @@ pipeline {
   }
   parameters {
     choice(name: 'runTestsSuite', choices: ['all', 'helm', 'ingest-manager', 'metricbeat'], description: 'Choose which test suite to run (default: all)')
+    booleanParam(name: "SKIP_GIT_CHECKS", defaultValue: "false", description: "If it's needed to check for Git changes to filter by modified sources")
```

Comment:
```suggestion
    booleanParam(name: "SKIP_GIT_CHECKS", defaultValue: false, description: "If it's needed to check for Git changes to filter by modified sources")
```

Reply:
Shouldn't this be the bool of `true` instead of the string?

Reply:
I copied it from apm-pipeline, and there almost all occurrences are quoted or double quoted.

Reply:
Thinking about it, I do prefer having boolean values without quotes. Removed them

Reply:
Ups, I did not realise there was a suggestion commit. My fault  😞